api/lib/utilities/ccdDataStoreApiUtilities.ts

```diff
@@ -30,21 +30,26 @@ const ERROR_UNABLE_TO_POST_CASE = 'Unable to POST case data using the Event Toke
  */
 export async function getEventToken(userId, caseId, jurisdiction, caseType, eventId) {
 
-    try {
-        const eventToken = await getEventTokenAndCase(
+    const response = await asyncReturnOrError(
+        getEventTokenAndCase(
             userId,
             jurisdiction,
             caseType,
             caseId,
             eventId
-        )
-        return eventToken.token
-    } catch (error) {
+        ),
+        `Error sending event`,
+        null,
+        logger,
+        false)
```

Comment:
if you remove the 'false' flag then asyncReturnOrError will send the correct status/response for you? could then remove the promise.reject below and just check for existence of response before returning the token

Reply:
Removed :)